src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -112,6 +112,7 @@ class ConfigurationHelper implements Serializable {
     Map use(){
         handleValidationFailures()
         MapUtils.traverse(config, { v -> (v instanceof GString) ? v.toString() : v })
+        if(config.verbose) step.echo "[${name}] Configuration: ${config}"
```

Comment:
Perhaps iterating and outputting each individual entry in a new line would make it more readable but as a debug output still sufficient.

👍